app/engine/penguin.py

```diff
@@ -47,8 +46,8 @@ def __init__(self, server: "SnowflakeWorld", address: IPv4Address | IPv6Address)
 
         self.member_card: MemberCard | None = None
         self.selected_card: CardObject | None = None
-        self.power_card_slots: List[CardObject] = []
-        self.power_cards: List[CardObject] = []
+        self.owned_cards: List[CardObject] = []
+        self.cards_ready: int = 0
```

Comment:
Please explain the differences between the previous card system and the new one. What does it help with?